examples/helloworld/lib/templates/hello/index.pug

```diff
@@ -0,0 +1,17 @@
+//-
+ Copyright 2016 Skytap Inc.
```

Comment:
Looks like this one got changed to 2016 instead of frames/index.pug?

Reply:
ahh thanks for catching that